controllers/clusterresources/redisuser_controller.go

```diff
@@ -132,9 +133,12 @@ func (r *RedisUserReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	}
 
 	for clusterID, event := range user.Status.ClustersEvents {
+		userID := fmt.Sprintf(instaclustr.RedisUserIDFmt, clusterID, username)
+
 		if event == models.CreatingEvent {
-			_, err = r.API.CreateRedisUser(user.Spec.ToInstAPI(password, clusterID, username))
-			if err != nil {
+			err = r.API.GetRedisUser(userID)
```

Comment:
It's strange that the Get method returns nothing. Let's return the user from the GetRedisUser method and validate that usernames are the same, wdyt?

Reply:
I think that we should return entity, but name validation unnecessary, because user ID contains user name, we can`t get user if it`s not equal
user ID example:        "id": "c1af59c6-ba0e-4cc2-a0f3-65cee17a5f37_myRedisUser"
User ID equals clusterID + "_" + username